composer.json

```diff
@@ -51,22 +51,14 @@
 		"ramsey/uuid": "^3.8",
 		"enshrined/svg-sanitize": "^0.13.0",
 		"guzzlehttp/guzzle": "^6.3",
-		"zendframework/zend-httphandlerrunner": "^1.1.0",
```

Comment:
@Uwe-Kohnle With this PR we'll remove `zendframework/zend-httphandlerrunner` (as discussed and decided).